aries/messages/src/msg_parts.rs

```diff
@@ -14,9 +14,9 @@ use typed_builder::TypedBuilder;
 /// `~attach` used in some messages that are in fact part of the protocol itself and are
 /// instrumental to the message processing, not an appendix to the message (such as `~thread` or
 /// `~timing`).
-#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, TypedBuilder)]
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, TypedBuilder, Display)]
 #[builder(build_method(vis = "", name = __build))]
-pub struct MsgParts<C, D = NoDecorators> {
+pub struct MsgParts<C, D> {
```

Comment:
Can you please specify the problem that the default type was causing?

Reply:
Prompted me to dig into this more - the issue was the proc macro was too simplistic, it was generating stuff like 
```
impl<C, D = NoDecorators> std::fmt::Display for MsgParts<C, D>
    where
        C: serde::Serialize,
        D: serde::Serialize,
```
while is should had been 
```
impl<C, D> std::fmt::Display for MsgParts<C, D>
    where
        C: serde::Serialize,
        D: serde::Serialize,
```
This is addressed now, syn crate has useful function `split_for_impl` available for `Generics` type, which allows to work with the generics related statements in more granular fashion.

So I am re-introducing the default NoDecorators type, reverting some of the changes done here

Reply:
I hope that the fact that the decision to rather modify the sites of usage instead of fixing the macro seemed as a better approach is more about due to skill issue rather than the fact that procedural macros are on harder to maintain :) Probably both, though...

Reply:
@mirgee I think you misunderstood me, I fixed the macro. No change to callsites anymore

Reply:
@Patrik-Stas No, I am talking about the initial decision.

Reply:
Ah yeah. In fact I misread. Indeed exactly as you said, it's definitely objectively better to make the macro more robust.